Orange/widgets/regression/owsgdregression.py

```diff
@@ -42,7 +40,7 @@ class OWSGDRegression(OWProvidesLearner, widget.OWWidget):
     learning_rate = settings.Setting(InvScaling)
 
     want_main_area = False
-    resizing_enabled = False
+    resizing_enabled = True
```

Comment:
While I understand your motivation here, this is not the right way to do it. Please revert this from this PR. If you feel like fixing this widget being too high, consider doing it by relayouting its contents into two columns.